.travis.yml

```diff
@@ -2,8 +2,6 @@ language: node_js
 sudo: false
 
 node_js:
-  - "6"
-  - "7"
```

Comment:
Is there any way we can polyfill promisify instead? Node 6 is an LTS release and still supported by Node for another year: https://github.com/nodejs/Release#release-schedule